builddirpath.py

```diff
@@ -14,15 +15,15 @@ def GetPath():
         elif choice != 'back':
             path.append(choice)
         for el in path:
-            cur_path+= '/' + el
+            cur_path += '/' + el
         if cur_path != '':
             print()
             print("cur path is: " + cur_path)
             print()
-            # print(os.listdir(cur_path))
+            nextpaths=[]
             for sub in os.listdir(cur_path):
                 subdir = os.path.join(cur_path, sub)
                 if os.path.isdir(subdir):
+                    nextpaths.append(sub)
                     print(sub)
-    return path
```

Comment:
if this is gone, what are we returning in this getter function?

Reply:
@Zalymo 
This script is a prototype that was developed to be turned into a view which is being worked on by @kevdouglass, the function doesn't need to return anything, it is just an infinite loop for a command line tool that I wrote as a proof of concept to allow a user to traverse directories and select a path, if the file is no longer needed as a reference then we can remove it from the repo.

Reply:
okay, that clears things up